tracer/src/Datadog.Trace/Configuration/ImmutableDynamicSettings.cs

```diff
@@ -20,6 +20,8 @@ internal class ImmutableDynamicSettings : IEquatable<ImmutableDynamicSettings>
 
         public bool? DataStreamsMonitoringEnabled { get; init; }
 
+        public bool? DataStreamsLegacyHeadersEnabled { get; init; }
```

Comment:
"Dynamic" settings are used for that is now called Remote Configuring. We don't need to add this here unless you're going to support changing this setting via remote config.

Reply:
👍 I think we don't need to add it to remote configuration.